### DISCUSSION TITLE
[Bug] it stops at connecting to Wallet

### DISCUSSION THREAD

**Discussion by ahmadzabihi5** [Chia-Network/chia-blockchain#10596] [Support]
### What happened?

Hi,
there is error when I want to load Chia software. it stops at connecting to Wallet in the first running of chia Software .
![Untitled](https://user-images.githubusercontent.com/100061790/154854220-574e4cfb-89b7-4030-8a42-bbe26dfbb04b.png)

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
it stops at connecting to Wallet
```

**Comment by Ruslan02538** [comment]
Не могу подключится к своему кошелку после отключения электричества

**Comment by esaung** [comment]
try to reboot.
